app/components/Modal/SendModal.jsx

```diff
@@ -244,11 +244,9 @@ class SendModal extends React.Component {
             this.setState(
                 {
                     from_name: np.from_name,
-                    from_account: ChainStore.getAccount(np.from_name),
+                    from_account: np.from_account,
```

Comment:
Why was this done?

Reply:
Sorry, I should have documented this change better. 

The component used to look up accounts (`<AccountSelector>`) returns this data from the following query on a change, so no need to query for the same data twice.

https://github.com/bitshares/bitshares-ui/blob/aa9b1f833e1a85c683f2131ec1af826bd06e5db7/app/components/Account/AccountSelector.jsx#L155-L162

It's then set in onFromAccountChange

https://github.com/bitshares/bitshares-ui/blob/aa9b1f833e1a85c683f2131ec1af826bd06e5db7/app/components/Modal/SendModal.jsx#L603-L605

https://github.com/bitshares/bitshares-ui/blob/aa9b1f833e1a85c683f2131ec1af826bd06e5db7/app/components/Modal/SendModal.jsx#L318-L320